src/http/server.clj

```diff
@@ -293,6 +294,24 @@
          (log/infof "\"%s %s HTTP/1.1\" %s %s" method'  uri status len))
        response))))
 
+(defn pw-prompt []
+  (str (do (println "Password: ") (.readPassword (System/console)))))
```

Comment:
`println` returns `nil`, so there's no practical need to execute it inside `str` call. It also makes code harder to read.

```
(defn ... []
  (println "...")
  (str (.readPassword ...))
```

looks better.

Reply:
Agree on your  change.  I had the `do` block because although `println` returns nil, the second form doesn't.  My prior version bound two symbols in a `let` block so i needed `do` to avoid 2 additional empty `_` bindings for `println` to prompt the user.  Now that's not needed, so your change looks much better.